pom.xml

```diff
@@ -452,11 +452,7 @@
                 <exclude>lang/ruby/.gem/**</exclude>
                 <exclude>lang/ruby/pkg/**</exclude>
                 <exclude>lang/ruby/.bundle/**</exclude>
-                <exclude>lang/rust/target/**</exclude>
-                <exclude>lang/rust/precommit_venv/**</exclude>
-                <exclude>lang/rust/Cargo.lock</exclude>
-                <exclude>lang/rust/README.tpl</exclude>
-                <exclude>lang/rust/.requirements-precommit.txt</exclude>
+                <exclude>lang/rust/README.md</exclude>
```

Comment:
@Xuanwo What do you use instead of RAT for license checking in your projects ?

Reply:
Hi, I'm using https://github.com/korandoru/hawkeye which written in rust and provides github actions natively:

```yml
licenses:
  runs-on: ubuntu-latest
  timeout-minutes: 10
  env:
    FORCE_COLOR: 1
  steps:
    - uses: actions/checkout@v4
    - name: Check license headers
      uses: korandoru/hawkeye@v5
```